src/utilities/device_utils.h

```diff
@@ -13,9 +13,11 @@
 /* Data types depending on GPU architecture */
 #if defined(HYPRE_USING_CUDA) || defined(HYPRE_USING_SYCL)
 typedef hypre_uint          hypre_mask;
+#define hypre_mask_one      1U
 
 #elif defined(HYPRE_USING_HIP)
 typedef hypre_ulonglongint  hypre_mask;
+#define hypre_mask_one      1ULL
```

Comment:
better to move this together with `HYPRE_WARP_FULL_MASK` and also change to all capitalized letters?

Reply:
Yes, agreed